bodhi/server/validators.py

```diff
@@ -1214,6 +1215,27 @@ def _validate_override_build(request, nvr, db):
     return build
 
 
+@postschema_validator
+def validate_eol_date(request, **kwargs):
+    """
+    Ensure the end-of-life date is in the right format.
+
+    Args:
+        request (pyramid.request.Request): The current request.
+        kwargs (dict): The kwargs of the related service definition. Unused.
+    """
+    eol_date = request.validated.get('eol')
+    if eol_date is None:
+        return
+
+    eol_date = str(eol_date)
+    if not re.match(r"20[0-9][0-9]-((0[1-9])|(1[0-2]))-(0[1-9]|[1-2][0-9]|3[0-1])", eol_date):
```

Comment:
This was meant as an additional date limitation, that for example date of 3021-12-12 or 1999-12-12 would not pass, because for releases it would probably not be relevant. I remember that at some point it was discussed if to allow dates in the past. I didn't add that check for the moment, in case we would want to insert eol dates for the past releases.

Reply:
I removed the regex and wanted to at least make sure that eol date will not be before release date. But I see the release date is not there :) Would it be nice to have the release date for releases?